scenarios/bigquery.py

```diff
@@ -101,19 +101,19 @@ def main(configs, project, bucket_path):
 
 if __name__ == '__main__':
     PARSER = argparse.ArgumentParser(description=
```

Comment:
done. I compared with and without and only noticed the description disappearing. Is displaying a description in the usage message bad for some reason?